### DISCUSSION TITLE
NOAA Weather Radio

### DISCUSSION THREAD

**Discussion by ChrisM510** [poblabs/weewx-belchertown#757] [Ideas]
I had an idea...  Does anyone else think it would be cool to be able to play your local NOAA Weather Radio station on your WeeWX + Belchertown site?  Maybe if we had a way of inputting a URL for a live audio feed?

Thoughts?

**Comment by roe-dl** [comment]
Am 11.01.22 um 15:07 schrieb ChrisM510:
> I had an idea...  Does anyone else think it would be cool to be able to play your local NOAA Weather Radio station on your WeeWX + Belchertown site?  Maybe if we had a way of inputting a URL for a live audio feed?
>
> Thoughts?
>

spoken forecasts:
https://beltoforion.de/de/wetterbericht/

**Comment by karlgo** [comment]
Sound like a good idea at least to make it available.

You might want to look at encapsulating the Broadcastify (https://www.broadcastify.com/listen/feed/27479) player into your site. Shouldn't be too hard since they provide a HTML 5 player.
